api/src/main/java/com/valtech/aem/saas/api/query/SimpleFilter.java

```diff
@@ -16,19 +15,36 @@
 public class SimpleFilter implements Filter {
 
     private static final String FILTER_FIELD_VALUE_DELIMITER = ":";
+    public static final String FILTER_STARTS_WITH_SUFFIX = "*";
```

Comment:
do we need to have this constant public?

Reply:
Used in FilterFactory as well, which is why I made it public (same goes for `NotFilter.FILTER_NOT_PREFIX`)
If you have a prettier way of doing this, I'm all ears :)

Reply:
aha ok. then just as a convinence:

1. move public constants before private ones
2. add minimal javadoc
afterwards you can resolve